internal/zinc-compile-core/src/main/scala/sbt/internal/inc/LoggerReporter.scala

```diff
@@ -53,7 +53,14 @@ object LoggerReporter {
   lazy val problemStringFormats: ProblemStringFormats = new ProblemStringFormats {}
 }
 
-class LoggerReporter(maximumErrors: Int, logger: ManagedLogger, sourcePositionMapper: Position => Position = { p => p }) extends Reporter {
+class LoggerReporter(
+  maximumErrors: Int,
+  logger: ManagedLogger,
+  sourcePositionMapper: Position => Position = { p =>
+    p
+  }
```

Comment:
Same as above, scalafmt doesn't seem to like single line curly brace lambdas at all 😅